_source/logzio_collections/_log-sources/lambda-extensions.md

```diff
@@ -0,0 +1,201 @@
+
+#### Deploying Logz.io logs extension via the AWS CLI
+
+<div class="tasklist">
+
+##### Deploy the extension, dependencies and configuration
+
+Use the following command to add the layer to your function and configure the environment variables:
+
+```shell
+aws lambda update-function-configuration \
+    --function-name <<FUNCTION-NAME>> \
+    --layers <<LAYERS>> \
+    --environment "Variables={<<ENV-VARS>>}"
+```
+
+**Note:** this command overwrites the existing function configuration. If you already have your own layers and environment variables for your function, list them as well.
+
+| Placeholder | Description |
+| --- | --- |
+| `<<FUNCTION-NAME>>` | **Required**. Name of the Lambda Function you'd like to monitor. |
+| `<<LAYERS>>` | A space-separated list of function layers to add to the function's execution environment. Specify each layer by its ARN, including the version. Check out the [Lambda extension versions table](#tables) for the ARN. You may need to add another layer that has the extensions dependencies. See the [extensions dependencies table](#tables) for the libraries that your extension requires. If your function doesn't already has those libraries under `/opt/python`, you'll need to add the dependencies as a layer, too. |
+| `<<ENV-VARS>>`  | Key-value pairs containing environment variables that are accessible from function code during execution. Should appear in the following format: `KeyName1=string,KeyName2=string`. For a list of all the environment variables for the extension, see the [environment variables table](#tables).|
+
+##### Run the function
+
+Run the function. Note that it may take more than one run of the function for the logs to start shipping to your Logz.io account.
+
+```shell
+aws lambda update-function-configuration \
+    --function-name <<FUNCTION-NAME>> \
+    --layers [] \
+    --environment "Variables={}"
+```
+
+<!-- info-box-start:info -->
+This command overwrites the existing function configuration. If you already have your own layers and environment variables for your function, include them in the list.
+{:.info-box.note}
+<!-- info-box-end -->
+
+
+</div>
+
+#### Deleting the extension
+
+To delete the extension, its dependencies and environment variables, use the following command:
+```shell
+aws lambda update-function-configuration \
+    --function-name some-func \
+    --layers [] \
+    --environment "Variables={}"
+```
+
+<!-- info-box-start:info -->
+This command overwrites the existing function configuration. If you already have your own layers and environment variables for your function, include them in the list.
+{:.info-box.note}
+<!-- info-box-end -->
+
+</div>
+<!-- tab:end -->
+
+<!-- tab:start -->
+<div id="console">
+
+#### Deploying Logz.io log extensions via the AWS Management Console
+
+<div class="tasklist">
+
+##### Add the extension to your Lambda Function
+
+1. In the Lambda Functions screen, choose the function you wish to monitor.
+[Insert screenshot 1-1]
```

Comment:
**Add the extension to your Lambda Function**: Missing screens 1 thru 3

Reply:
I'll talk to you about it in our meeting :)